node/service/src/lazy_loading/mod.rs

```diff
@@ -656,6 +657,12 @@ where
 								)),
 								Some(UpgradeGoAhead::GoAhead).encode(),
 							));
+							additional_key_values.push((
+								relay_chain::well_known_keys::upgrade_restriction_signal(
+									ParaId::new(parachain_id),
+								),
+								None::<UpgradeRestriction>.encode(),
+							));
```

Comment:
I just realised that we could move this code to `additional_key_values`. But not important